src/ClientData/include/ClientData/DataManager.h

```diff
@@ -29,6 +29,18 @@
 
 namespace orbit_client_data {
 
+// TimeRange represents an inclusive time range. A timer is only considered within the time range if
+// it is fully enclosed in it.
+struct TimeRange {
+  TimeRange(uint64_t start, uint64_t end)
+      : start(std::min(start, end)), end(std::max(start, end)) {}
+  [[nodiscard]] bool IsTimerInRange(const orbit_client_protos::TimerInfo& timer) const {
+    return timer.start() >= start && timer.end() <= end;
```

Comment:
Nit. 
In multiple comparisons glued together with && I prefer to have the arguments ordered under assumption that it computes to true. Like  `start <= timer.start() && timer.end() <= end`.

Reply:
done